Detections/SlackAudit/SlackAuditEmpty UA.yaml

```diff
@@ -0,0 +1,24 @@
+id: 45ea3a67-2837-4584-9a06-de5881e640da
+name: SlackAudit - Empty User Agent
+description: |
+  'This query helps to detect who trying to connect to the Slack Workspace with Empty User Agent.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: SlackAuditAPI
+    dataTypes:
+      - SlackAudit_CL
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics: Persistence
+query: |
+  SlackAudit
+  | where isempty(UserAgentOriginal)
+  | project SrcUserName, SrcIpAddr, DvcAction
```

Comment:
I am not entirely familiar with Slack logs, but based on the parser, it seems like the analyst receiving these alerts could benefit from more fields being brought thru in the project statement.

Reply:
Fixed, Pls check